src/index.js

```diff
@@ -3,8 +3,21 @@
  * @author E-Com Club <ti@e-com.club>
  * @license AGPL-3.0
  */
+import EcomAuth from '@ecomplus/auth'
+import fetchApplications from './methods/fetch-applications'
+import findApplications from './methods/find-applications'
+import installApplication from './methods/install-application'
+import removeApplication from './methods/remove-application'
+import patchApplication from './methods/patch-application'
 
-const EcomApps = function () {
+const EcomApps = function (sessionKey) {
+  const self = this
+  this.ecomAuth = new EcomAuth(sessionKey)
+  this.fetch = (meta = {}) => fetchApplications(meta)
+  this.find = appId => findApplications(appId)
+  this.install = appId => installApplication(self, appId)
+  this.remove = _id => removeApplication(self, _id)
+  this.patch = (_id, { data, hiddenData }) => patchApplication(self, _id, { data, hiddenData })
```

Comment:
acho que você está colocando nomes de métodos muito simplificados, não? 

```suggestion
  this.editApp = (_id, body) => patchApplication(self, _id, body)
```

Reply:
neste método também sugiro que você receba o body todo do application, tendo a propriedade `data` e/ou `hidden_data` você bate o(s) respectivo subresource, tendo outras propriedades (como `state` por exemplo) você bate o resource `applications`, entendeu? o que acha?

Reply:
Acho ótimo!